File: client/Client/Network/Messages/Session/AcceptUserLoginMessage.cs
namespace LoadRunnerClient.Messages
{
    public class AcceptUserLoginMessage
    {
        public const string TYPE = "AcceptUserLoginMessage";

        public AcceptUserLoginMessage()
        {
        }

        private bool _dummy;
        public bool dummy { get => _dummy; set => _dummy = value; }
    }
}

File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/user/AcceptUserLoginMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.user;

public class AcceptUserLoginMessage {

	public static final String TYPE = "AcceptUserLoginMessage";

	private boolean dummy;

	public AcceptUserLoginMessage() {
	}

	public AcceptUserLoginMessage(boolean dummy) {
		this.dummy = dummy;
	}

	public boolean isDummy() {
		return dummy;
	}

	public void setDummy(boolean dummy) {
		this.dummy = dummy;
	}
}
